src/chart/generateCategoricalChart.tsx

```diff
@@ -1200,8 +1269,8 @@ export const generateCategoricalChart = ({
 
         const newState = {
           ...defaultState,
-          ...keepFromPrevState,
           ...updatesToState,
+          ...keepFromPrevState,
```

Comment:
Why this change?

Reply:
It was to help manage if the tooltip should be updated when the image gets resize. I'll rewrite it for clarity.

Reply:
It looks suspicious because it switched the order of overrides. "Keep" and "updates" are now in the opposite order of what the object destructuring does. Are we 100% certain there are no unintended side effects?